programs/eosio-launcher/main.cpp

```diff
@@ -809,6 +809,7 @@ launcher_def::define_network () {
         if (lhost) {
           bindings.emplace_back(move(*lhost));
           delete lhost;
+          lhost = nullptr;
```

Comment:
I think it would be better to change this to use a `std::optional`

Reply:
Decided to make it a std::unique_ptr.